mmpose/core/evaluation/top_down_eval.py

```diff
@@ -250,6 +250,83 @@ def _taylor(heatmap, coord):
     return coord
 
 
+def post_dark(coords, batch_heatmaps, share_heatmap=True):
+    """DARK post-pocessing.
+
+    Note:
+        batch_size: N
+        num_keypoints: K
+
+    Args:
+        coords (np.ndarray[N, K, 2]): Coords in shape of batchsize*num_kps*2.
+        batch_heatmaps (np.ndarray[N, K, H, W]): batch_heatmaps
+        share_heatmap (bool) : True for bottom_up paradigm where all persons
+                    share the same heatmap while False for top_down paradigm
+                    where each person has its own heatmaps.
+
+    Returns:
+        res (np.ndarray): Refined coords in shape of batchsize*num_kps*2.
+    """
+    if not isinstance(batch_heatmaps, np.ndarray):
+        batch_heatmaps = batch_heatmaps.cpu().numpy()
+    batch, num_kps, h, w = batch_heatmaps.shape
+    persons = coords.shape[0]
+    if share_heatmap:
+        assert batch == 1
+    else:
+        assert batch == persons
+    for i in range(batch):
+        for j in range(num_kps):
+            mapij = batch_heatmaps[0, j, :, :]
+            mapij = cv2.GaussianBlur(mapij, (3, 3), 0)
+            batch_heatmaps[0, j, :, :] = mapij
```

Comment:
cv2 generally supports inplace operations

Try
```python
    for i in range(N):
        for j in range(K):
            mapij = batch_heatmaps[i, j]
            cv2.GaussianBlur(mapij , (3, 3), 0, mapij)
```

Reply:
Or
```python
    for heatmaps in batch_heatmaps:
        for heatmap in heatmaps :
            cv2.GaussianBlur(heatmap, (3, 3), 0, heatmap)
```

Reply:
yes, cv2 support this and the latter is adopted